### ISSUE TITLE
Have you considered an Ansible-Nas organization?

### ISSUE THREAD

**Issue by Narcolapser** [davestephens/ansible-nas#371]
**Is your feature request related to a problem? Please describe.**
You have 62 open issues (now 63) and 55 open pull requests (many fixing those open issues, just waiting to be merged). You seem to have a little more here then you are able to comfortably handle. 

**Describe the solution you'd like**
Setup an Ansible-Nas github organization and get some of the people who contribute the most to Ansible-Nas or who you otherwise trust to assist in merging the pull requests.

**Describe alternatives you've considered**
The alternative would be the 217 forks. That's a lot of forks, most of them dead ends that could have been improvements. It would be better to focus effort on to your one single repository.

**Additional context**
I have done this. I started the python-O365 project years ago and then lost access to an Office365 account. Setup an organization and another user who was still using it has more or less taken the project over from me. It's a great way to continue the growth of your project without being solely dependent upon you. Judging by the open pull requests @bcurran3 would be a good candidate. I can help if you'd like, but I'd totally understand not; as you have zero reason to trust me at this point.

**Comment by PurpleNinja225** [comment]
I second this. 

I have used ansible-nas as a teaching tool to help get me more familiar with docker, ansible, and containerizing services. I have also started using it in my business (with some extra addons like mail, ad, oauth, etc.) I have been interested in adding content and features to the project as I now have a vested interest in the continuation of it. 

However, I have 2 issues keeping me from that, 

1. I'm pretty new to contributing to github :sweat_smile: and I'm nervous of changing things to be submitted. 

2. I have upgraded my installation to use traefik2.2, which required modifying a lot of files, so I don't want to start adding other content until we can get at least that container upgraded. I'm okay with showing my full config to @davestephens if it could help speed things along and I have already shared my traefik.yml and .toml in #142

**Comment by Narcolapser** [comment]
Any thoughts on this @davestephens?

**Comment by davestephens** [comment]
Thanks for your thoughts! I think your question is more about maintenance of the project rather than a github organisation specifically - outstanding open issues and merging PRs are mutually exclusive to whether ansible-nas is under a github org or not. 

The only thing that's truly blocked on me is merging of PRs and closing issues, and I'd like to keep it that way until I find people willing to contribute that also have sufficient linux and ansible knowledge to help maintain the project. 

Another thing to bear in mind is that users of ansible-nas are very transient, and someone super-enthusiastic right now quite possibly won't be a month or two later. They get their home server set up and won't be seen again, because for the most part - an ansible-nas installation maintains itself once it's set up.

**Comment by bcurran3** [comment]
I'm still here.

**Comment by Narcolapser** [comment]
tl;dr: get some help, you are giving the impression you project isn't maintained. 

Setting up an organization is about more than just adding collaborators. I see you can just add collaborators to a repo directly (news to me). Setting up an organization makes your project more appealing, it says "hey, this is bigger than me, and should I disappear it will carry on." 

Yes, the only thing that is blocked on you are the issues and PRs, but considering my PR has been sitting for 19 days on what is basically a yes or no question you don't seem to be following it closely. What this interaction and the interaction on my PR is suggesting to me is that you don't actively maintain this project very much. Infact most of the PRs and issues would suggest a maintainer that isn't paying attention. For example #244, an issue where the user is simply asking you to merge some pull requests. They may be suitable to the project but you haven't given any indication either way. or #357 where you requested a change, the user made it, but you haven't merged it, 3 months later. Looks like @sshestopalov got tired of waiting, forked your project and merged it himself. It makes me think that I, or @bcurran3, or @PurpleNinja225 should fork and carry on without you. I don't like forking, I think the open source community could use with a little less forking, but it is useful when we have reached an impasse. 

You already have a presence, you show up in searches, that is how I found the project and don't want to leave that behind with a fork. While I understand that most of the community is going to be transient, there are people, like myself and (i assume) @bcurran3 who enjoy working on these sorts of projects. I think you will find that more of your community would be willing to stick around and help if their PRs got merged faster.

**Comment by davestephens** [comment]
As above - your comments are based on your opinion of my maintenance of the project rather than whether the code is in an organisation or not. 

Just so we're on the same page - this is a unpaid project I do for fun on the side. Note the key word - "**fun**".  It is not a business, my full time job, nor have I committed to maintaining any sort of SLA for its users. As a laugh I checked how much I've earnt since starting the project - £123 - £30/year. Additionally, the same as most people on earth, I have a day job, a family to support, as well as personal things that meant I have had zero time for anything in the "fun" category in the last year.

This isn't intended to be a "woe is me" comment, but I think people conveniently forget that there are real people with real lives behind GitHub projects when something they want isn't happening fast enough, despite the fact they've already got a piece of software for free, and that some are even making money from. Pointing out how you're unhappy about x, y and z, and how a, b and c must happen, without considering that, sounds very entitled.

Moving forward, you'll notice that I have time to spend on the project again, and am working my way through PRs and issues, as well as numerous enhancements.

**Comment by PurpleNinja225** [comment]
Would creating a dev branch and allowing some contributor access be an option? So those who really want to help put time and effort into the project can contribute without affecting the main branch that everyone uses and only merge when ready. 
_____________________________________________________________________________________________________

In response to @davestephens, I know your comment wasn't directed exactly at me, but in my experience I understand that projects like this are labors of love, and the reason I ask to add collaborators is because I'm someone who can get paid to work on this project. As I mentioned in my above comment, my work uses ansible-nas, and I can help add new apps, like vpn clients (not just for transmission), SSO, Oauth, AD, making more configuration settings for people who set up multiple instances ( Use it both at work and at home).

I don't say this because I wan't to be added as a contributor, i'm not confident enough in my code etiquette to. But opening things up to someone like @bcurran3 who has made 114 commits to this project! 

In reference to @Narcolapser I do have a fork but its private, as it has a ton of code changes, and is for work :sweat_smile: 

I'm really happy to see new commits since the last one before that was like sept.? It would be nice to let people help when they can while they do have that passion for this project, cause if they have to wait to long they fix it for just themselves and then move on, that's what I did. Plus having an extra pair of eyes that can get on when you aren't able to only benefits the following.

**Comment by Narcolapser** [comment]
I am totally tracking with you on "this isn't my Job". When I maintained O365 I felt the same. But I am not asking you to do more, in theory, I'm asking you to do less. Ignore the whole organization bit, my real question is "need some help?"

**Comment by davestephens** [comment]
Always happy to receive help! :-) I'm just not comfortable giving anyone commit/merge access to `master` just yet. It's not you, it's me ;-) 

I'm going to close this for now as I'm not sure there's anything more to add. I like to think that when the right people come along with the right set of skills and commitment, we'll all know (and I'm not saying that's *not* any of you guys, but you're all reasonably new to the project).

**Comment by Narcolapser** [comment]
> I'm not saying that's not any of you guys, but you're all reasonably new to the project).

Agreed.

Well, I hope it works out. This is a neat project, hopefully you'll have the motivation to maintain it until those right people come along!
